### PR TITLE
Added option for sticky header

### PR DESCRIPTION
**Pull request by jweston491**
Added option in `customizer.php` for Fixed Header. SCSS and JavaScript to account for margin offset when logged in and scroll position.

### CODE REVIEW

File: inc/customizer.php [hunk 1]
```diff
@@ -67,6 +67,31 @@ function understrap_theme_slug_sanitize_select( $input, $setting ) {
 
 		}
 
+		// Sticky Header options
+		$wp_customize->add_setting(
+			'understrap_header_position',
+			array(
```
Comment (pattonwebz) [inline inc/customizer.php]: This is missing a `sanitize_callback`. You could use this for one: 

https://github.com/WPTRT/code-examples/blob/master/customizer/sanitization-callbacks.php#L23

File: header.php [hunk 2]
```diff
@@ -31,7 +31,7 @@
 
 		<a class="skip-link sr-only sr-only-focusable" href="#content"><?php esc_html_e( 'Skip to content', 'understrap' ); ?></a>
 
-		<nav class="navbar navbar-expand-md navbar-dark bg-primary">
+		<nav class="navbar navbar-expand-md navbar-dark bg-primary <?php echo (!get_theme_mod('understrap_header_position')) ?: 'fixed-top' ?>">
```
Comment (pattonwebz) [inline header.php]: Missing the ending semi-colon and also some PHPCS formatting adjustments:

`echo  ( ! get_theme_mod( 'understrap_header_position' ) ) ?: 'fixed-top';`

File: src/js/sticky-header.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+//Fixed navbar
```
Comment (pattonwebz) [inline src/js/sticky-header.js]: Could there be more comments in this file? File header comments for sure but also inline comments would be use too :)

**Comment by pattonwebz** [comment]
Hey @jweston491,

This seems like a nice addition to me. I have not tested the code but I did read through it and offered a couple of small review comments with suggestions. Thanks you for the PR :D

**Comment by jweston491** [comment]
Thanks! Getting better at coding every day. Incorporated your suggestions and added some CSS for when using the Customizer.

**Comment by IanDelMar** [comment]
One suggestion: Go for the class `admin-bar` instead of `logged-in`. A logged in user might have set the admin bar to be hidden on front.

**Comment by Thomas-A-Reinert** [comment]
I know that @holger1411 will disagree on integrating any changes to the customizer because every "_real developer would know how to realize it and to keep the theme bare-bone_" to my experience.
Personally, I´d got for it, @jweston491 . So I´ll give it some days to be confirmed by holger and if nothing happens, I´ll merge it, okay?

**Comment by Thomas-A-Reinert** [comment]
Oh, wow. "_Approved_" like almost 5 months ago. With conflicts. And no merge.
@pattonwebz @IanDelMar @0dp @GuerrillaCoder @noelspringer @MarieComet - whaya think?

**Comment by IanDelMar** [comment]
Well, actually, I don't know what the approval by @UnderstrapFramework means. There were a couple of approvals not leading to merges as far as I remember. As I don't think that anyone has tested this thoroughly maybe it's best that it didn't happen :-D But to be honest, I do not care a lot anymore.

**Comment by UnderstrapFramework** [comment]
Sorry I think I meant to just close it

**Comment by Thomas-A-Reinert** [comment]
I guess this was all caused by a [language barrier](https://github.com/understrap/understrap/issues/1071#issuecomment-584573404).. There seem to a  lot these days.

**Comment by MarieComet** [comment]
Hey, I don't know why I'm notified here again.
I've read some (long) issues. What is happening here ?
It looks like the mood has changed a lot, not positively. It's a shame and sad for longtime contributors.
@0dp @Thomas-A-Reinert (and others?), maybe it's time to move to another project, or create something better with (for) UnderStrap, don't you think ?
Good luck anyway!

**Pull request closed without merging**